### PR TITLE
fix: rename rmrk

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Fix
- [ ] Feature
- [ ] Refactoring
- [x] Chore

### What's new?

- [x] PR closes kodadot/nft-gallery#4841
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR
- [ ] I've written some unit tests 🧪

### Optional

- [x] I found edge cases

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### CODE REVIEW

**Comment by Jarsen136** [comment]
Hey, @vikiival Could you pls publish it after PR is merged?

**Review by vikiival — CHANGES_REQUESTED** [review]
Please leave single quotes

File: vuex-options/src/defaults/prefixes.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const PREFIXES: Option[] = [
   },
   {
     info: 'kusama',
-    text: 'RMRK',
+    text: 'Kusama',
     value: 'rmrk'
```
Comment (vikiival) [inline vuex-options/src/defaults/prefixes.ts:17]: 👀 and this sir?

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Please bump version

**Comment by Jarsen136** [comment]
> Please bump version

PR: https://github.com/kodadot/packages/pull/97
Let's go
